### DISCUSSION TITLE
Add layer support for `Fossify Gallery`

### DISCUSSION THREAD

**Discussion by nkuk377** [FossifyOrg/General-Discussion#263] [Ideas]
Hi
I wanted to thank to all of this project developers for your time and effort!
I used `SImple Mobile Tools` until they dropped F-Droid store. I moved to Fossify tools and I wanted to post an idea might be useful for users.
I used `Pocket Paint` for adding text (typed not drawn) to pictures, but at some point it started to crash (GrapheneOS), I discovered that it's abandoned as there was no update for more than 1 year, I had to remove it.
I was unable to find any lightweight image editor capable to add a text on the pictures, not on F-Droid store which could replace the app I mentioned.
I wanted to suggest to add this functionality to your `Fossify Gallery` if possible as it already has image editing options.
It could be the one of a kind app with this function on F-Droid and not just there, on Play Store too.
Thanks

**Comment by Aga-C** [comment]
Adding text to images is already requested here: https://github.com/FossifyOrg/Gallery/issues/97.

Next time, if you want to ask about a specific app, please use discussions in that app's repository.
